Altis_Life.Altis/core/shops/fn_chopShopSell.sqf

```diff
@@ -2,30 +2,28 @@
 /*
     File: fn_chopShopSell.sqf
     Author: Bryan "Tonic" Boardwine
+    Modified: Casperento
 
     Description:
     Sells the selected vehicle off.
 */
 disableSerialization;
-private ["_control","_price","_vehicle","_nearVehicles","_price2","_chopable"];
-_control = CONTROL(39400,39402);
-_price = _control lbValue (lbCurSel _control);
-_vehicle = _control lbData (lbCurSel _control);
+
+private _control = CONTROL(39400,39402);
+private _price = _control lbValue (lbCurSel _control);
+private _vehicle = _control lbData (lbCurSel _control);
 _vehicle = call compile format ["%1", _vehicle];
```

Comment:
Would probably be best to remove this

Reply:
i would personally use setVariable, instead of lbDate when using anything but a string.

Reply:
Thx for suggesting